firefox-ios/Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -452,6 +450,21 @@ class HistoryPanel: UIViewController,
         return cell
     }
 
+    private func getGroupCell(searchTermGroup: ASGroup<Site>, indexPath: IndexPath) -> UITableViewCell? {
+        guard let cell = tableView.dequeueReusableCell(
+            withIdentifier: TwoLineImageOverlayCell.cellIdentifier,
+            for: indexPath
+        ) as? TwoLineImageOverlayCell else {
+            logger.log("History Panel - cannot create TwoLineImageOverlayCell for STG",
```

Comment:
Can you change STG -> Search Term Group 

I think this would be more clear for debugging.